src/epoch.rs

```diff
@@ -29,41 +29,45 @@ impl Epoch {
 
     /// Attempts to advance the global epoch.
     ///
-    /// The global epoch can advance only if all currently pinned mutators have been pinned in the
-    /// current epoch.
+    /// The global epoch can advance only if all currently pinned participants have been pinned in
+    /// the current epoch.
     ///
     /// Returns the current global epoch.
     ///
     /// `try_advance()` is annotated `#[cold]` because it is rarely called.
     #[cold]
-    pub fn try_advance<'scope>(&self, registries: &List<LocalEpoch>, scope: &Scope) -> usize {
+    pub fn try_advance<'scope>(
+        &'scope self,
+        registries: &'scope List<LocalEpoch>,
+        scope: &'scope Scope,
+    ) -> usize {
         let epoch = self.epoch.load(Relaxed);
         ::std::sync::atomic::fence(SeqCst);
 
-        // Traverse the linked list of mutator registries.
+        // Traverse the linked list of participant registries.
```

Comment:
Nit: We could address this later, but -- a *registry* is a place where *registers* or *records* are kept.

More concretely, our linked list is the *registry*, and we usually refer to our *records* in the *registry* (or *entries* in the linked list) as *participants*. :)

The `for registry in registries.iter(scope)` loop would read better like this:
`for participant in registry.iter(scope)`